community/slack.md

```diff
@@ -1 +1,69 @@
-# Slack
+# Slack - Day to Day communication
+We use Slack to communicate both as paid staff and within our community organization. Slack is a valuable tool for both asynchronous, synchronous, and remote communication.
+
+We have automatic feeds setup in Slack to track various GitHub and Discourse activities as follows:
+
+* **#discourse-updates:** this channel flags new Discourse posts so that we can keep track of them in a single place. This channel also announces a new package when it’s posted in the Discourse pyOpenSci packages forum. The bot was built and maintained by Chiara Marmo.
+* **#feed-software-review:** this channel tracks new review submissions in our software-submission GitHub repo. This feed helps pyOpenSci staff and its editorial team keep track of the cadence of submissions in a centralized location.
+* **#github-issues:** this channel was created to make it easier to follow issues and pull requests opened in our core Github repositories, as given the volume of activity it’s easy to miss new items.
+
+## Setting up a Slack bot using Slack Workflows
+Administrators in the pyOpenSci Slack space can set up [workflows](https://slack.com/help/articles/16583775096083-Automations--What-is-a-Slack-workflow), which are automated bots that complete tasks based on automatic triggers. We currently use a workflow to onboard new members to the Slack space, and if there are additional bots you feel would be helpful, please reach out to Leah or Jesse.
+
+## Setting up an automatic feed to post from GitHub into Slack:
```

Comment:
Here are a few note

* The slack integration settings [can be found here](https://github.com/pyOpenSci/software-submission/settings/installations) in GitHub. IF YOU HIT THE configure button, it will allow you to configure which repositories are available to the slack / github integration bot. 
<img width="880" alt="Screenshot 2024-05-30 at 2 46 37 PM" src="https://github.com/pyOpenSci/handbook/assets/7649194/480a1468-4056-4146-8cec-0db800301e71">

@kierisi the part that i can't remember here is how i initially set this up. It may have been by calling a command in slack which lead me to the initial setup. Or i may have [started on this website](https://slack.github.com/) where you can add the bot to an organization? 

does that help? the commands below that you wrote out look correct to me.

Reply:
@lwasser that helps! it looks like the link you've shared is the way to go in order to set up the integration for the overall slack workspace, so I can update everything!